docs/reference/data-streams/downsampling.asciidoc

```diff
@@ -0,0 +1,171 @@
+<<ilm-downsample,Downsample action>> in your ILM policy and set `fixed_interval`
+to the level of granularity that you'd like:
+
+```
+PUT _ilm/policy/my_policy
+{
+  "policy": {
+    "phases": {
+      "warm": {
+        "actions": {
+          "downsample" : {
+            "fixed_interval": "1h"
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+[discrete]
+[[querying-downsampled-indices]]
+=== Querying downsampled indices
+
+You can use the <<search-search,`_search`>> and <<async-search,`_async_search`>>
+endpoints to query a downsampled index. Multiple raw data and downsampled
+indices can be queried in a single request, and a single request can include
+downsampled indices at different granularities (different bucket timespan). That
+is, you can query data streams that contain downsampled indices with multiple
+downsampling intervals (for example, `15m`, `1h`, `1d`).
+
+The result of a time based histogram aggregation is in a uniform bucket size and
+each downsampled index returns data ignoring the downsampling time interval. For
+example, if you run a date_histogram aggregation with `"fixed_interval": "1m"`
+on a downsampled index that has been downsampled at an hourly resolution
+(`"fixed_interval": "1h"`), the query returns one bucket with all of the data at
+minute 0, then 59 empty buckets, and then a bucket with data again for the next
+hour.
+
+NOTE: When runnning queries in {kib} and through Elastic solutions, a normal
+response is returned without notification that some of the queried indices are
+downsampled.
+
+[discrete]
+[[downsampling-restrictions]]
+=== Restrictions and limitations
+
+Note the following restrictions and limitations for downsampling:
+
+* Only indices in a <<tsds,time series data stream>> are supported. 
+
+* Data is downsampled based on the time dimension only. All other dimensions are
+copied to the new index without any modification.
+
+* Within a data stream, a downsampled index replaces the original index and the
+original index is deleted. Only one index can exist for a given time period. 
+
+* A source index must be in read-only mode for the downsampling process to
+succeed. Check the <<downsampling-manual,Run downsampling manually>> example for
+details.
+
+* Downsampling data for the same period many times (downsampling of a
+downsampled index) is supported. The downsampling interval must be a multiple of
+the interval of the downsampled index.
+
+* Downsampling is provided as an ILM action. See <<ilm-downsample,Downsample>>.
+
+* The new, downsampled index is created on the data tier of the original index
+and it inherits its settings (for example, the number of shards and replicas).
+
+* The numeric `gauge` and `counter` <<mapping-field-meta,metric types>> are
+supported.
+
+* The downsampling configuration is extracted from the time series data stream
+<<tsds-create-mappings-component-template,index mapping>>. The only additional
+required setting is the downsampling `time_interval`.
```

Comment:
```suggestion
required setting is the downsampling `fixed_interval`.
```

Reply:
Fixed in [507419d](https://github.com/elastic/elasticsearch/pull/88571/commits/507419dcc6e4498ad50f37505d150e5de4db8a77)